docs/how-to-monitor-and-manage.md

```diff
@@ -82,6 +116,12 @@ zk_open_file_descriptor_count: current opening fd count
 zk_max_file_descriptor_count: max opening fd count
 zk_followers: follower count, only present on the leader
 zk_synced_followers: synced follower count, only present on the leader
+apply_read_request_time_ms: The time for request processor to process read request
```

Comment:
The time for request processor to process read request, log replication is not included for write request.